### PR TITLE
Add variable token precision for invariant tests

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
Add the ability for Invariant testing with multiple combination of different precisions of Quote and collateral tokens.

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
LGTM

File: test-invariant-multi-precision.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+for quote_precision in 6 8 18
```
Comment (grandizzy) [inline test-invariant-multi-precision.sh]: Should we make target for this and rename existing so we'll have `make test-invariant-erc20`
`make test-invariant-erc20-precision`
and same for erc721?

Comment (prat-gpt) [reply test-invariant-multi-precision.sh]: this would be better. Will update.

Comment (prat-gpt) [reply test-invariant-multi-precision.sh]: updated here [2ca24ab](https://github.com/ajna-finance/contracts/pull/709/commits/2ca24ab17ed596703f0f34a48745f55bda03da34)

**Review by grandizzy — APPROVED** [review]

**Pull request merged**